scipy/stats/_continuous_distns.py

```diff
@@ -5392,8 +5392,165 @@ def _updated_ctor_param(self):
         return dct
 
 
+def _get_shape_parameters(distribution):
+    """
+    Returns all shape parameters of a scipy distribution
+    including loc and scale
+    @param distribution scipy.stats.rv_continuous
+    """
+    shape_parameters = []
+    if distribution.shapes is not None:
+        shape_parameters += distribution.shapes.split(', ')
+    shape_parameters += ['loc', 'scale']
```

Comment:
Maybe to given an example, in line 5512 of the same file I call the pdf method of a distribution like this. 
 distribution.pdf(x, *shape)

The last two elements of the shape variable must be loc and scale.